### DISCUSSION TITLE
how can get all check rule list and related description?

### DISCUSSION THREAD

**Discussion by dongshen** [actiontech/sqle#2344] [Q&A]
how can get all check rule list and related description?
is there some command get the list ？ or is defined in some files?

**Comment by sjjian** [comment]
可以通过平台导出规则模版，不过文件是json格式。你可以想办法将它转成excle，例如通过在线转换的网站
<img width="1439" alt="image" src="https://github.com/actiontech/sqle/assets/18008452/37f319fe-c16b-4241-a5fe-31bcf1f56fb3">
